builtin/trap_osh.py

```diff
@@ -110,6 +113,20 @@ def _RemoveUserTrap(self, sig_num):
             # shells do.
             iolib.sigaction(sig_num, SIG_DFL)
 
+    def _IgnoreSignal(self, sig_num):
```

Comment:
Hm this is very similar to `_RemoveUserTrap()` .... I wonder if we should consolidate them